projects/netgrif-components-core/src/lib/header/abstract-header-service.ts

```diff
@@ -81,7 +81,11 @@ export abstract class AbstractHeaderService implements OnDestroy {
     }
 
     get overflowMode(): boolean {
-        return this._overflowService.overflowMode;
+        if (this._overflowService !== null) {
```

Comment:
What if `_overflowService` is not null, but undefined?

Cannot be simplified to `!!this._overflowService`?

Reply:
Done